public_html/side/sidebarsnew.css

```
@@ -0,0 +1,68 @@
+body {
+    min-height: 100vh;
+    min-height: -webkit-fill-available;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)